forms/Coupon.js

```diff
@@ -77,8 +78,15 @@ class CouponForm extends React.Component {
 
 	constructor(props) {
 		super(props)
+        this.state = {
+            couponUsers: []
+        }
 	}
 
+    onUserSelected = (newCouponUsers) => {
```

Comment:
Change this to 

`onUserSelected = (couponUsers) => {
        this.setState({couponUsers})
    }`

Reply:
Notice the callback function name. Should be plural. not singular